script/test

```diff
@@ -11,6 +11,6 @@ export NIX_PATH="nixpkgs=./nix"
 echo "Building"
 
 # Build and create a root
-nix-build --no-link
+nix-build --sandbox --no-link --max-jobs 10
```

Comment:
Does it work with the default Nix installation (sanboxing is not available)?

Reply:
What do you mean default Nix installation? On both Linux and macOS sandbox works for me by using the default installer (and nothing more) (albeit with some tweaks on macOS).

We do want to test that everything works with the sandbox; if someone doesn't have support for the sandbox they can simply run `nix-build`.

What do you think?

Reply:
I though sandbox was not working by using the default installer. So, if it is working by default, that's fine with me.